channels/sg.m3u

```diff
@@ -9,3 +9,5 @@ https://d2e1asnsl7br7b.cloudfront.net/7782e205e72f43aeb4a48ec97f66ebbe/index_5.m
 https://i.mjh.nz/au/Sydney/tv.cna.m3u8
 #EXTINF:-1 tvg-id="MiaoMi.sg" tvg-name="妙米" tvg-country="SG" tvg-language="Malay" tvg-logo="https://www.mewatch.sg/blob/12295542/7cbb9cb1e909e0ba2230d2aebcefba7e/ico-miaomi-v2-data.png" group-title="",妙米
 https://liveanevia.mncnow.id/live/eds/MiaoMi/sa_dash_vmx/MiaoMi.mpd
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://www.asiantvawards.com/wp-content/uploads/2017/08/tvN-movie-logo-no-tagline.jpg" group-title="",tvN Movies
+https://liveanevia.mncnow.id/live/eds/tvNMovies/sa_dash_vmx/tvNMovies.mpd
```

Comment:
tvN Movies is Korean channel: https://www.lyngsat.com/tvchannels/kr/TVN-Movies.html